### ISSUE TITLE
CI fails if test file contains the word "unexpected" or "unresolved"

### ISSUE THREAD

**Issue by CohenArthur** [Rust-GCC/gccrs#2386]
https://github.com/Rust-GCC/gccrs/pull/2373 is failing because @MahadMuhammad added a testfile named "unexpected_arguments.rs". This is a funny behavior but it would be nice not to have it haha.

https://github.com/Rust-GCC/gccrs/actions/runs/5465722215/jobs/9949574002?pr=2373

**Comment by CohenArthur** [comment]
This is also happening in https://github.com/Rust-GCC/gccrs/pull/2383

**Comment by tschwinge** [comment]
Yeah, we should use a better method for detecting regressions than the current `grep unexpected` etc.

..., but I don't know yet how to properly do that.  (I always `diff` before/after `*.sum` files, but that's not applicable for this CI system.)

**Comment by dkm** [comment]
> Yeah, we should use a better method for detecting regressions than the current `grep unexpected` etc.
> 
> ..., but I don't know yet how to properly do that. (I always `diff` before/after `*.sum` files, but that's not applicable for this CI system.)

There's also the `./contrib/dg-cmp-results.sh` that is nice for this task.

As a bandaid, we could match `# unexpected ...`, as it's done in the bootstrap workflow. Or is there a reason why we can't?

**Comment by CohenArthur** [comment]
Yeah I think we could definitely look for `# of {unexpected,unresolved}` instead. Can't think of a reason why not
